conan/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, CMake, tools
```

Comment:
I am not sure about how conan works exactly, but I'm pretty sure this file should be on the CCI instead.